internal/notification/smtp_notifier.go

```diff
@@ -156,6 +156,12 @@ func (n *SMTPNotifier) dial() (err error) {
 
 // Do startTLS if available (some servers only provide the auth extension after, and encryption is preferred).
 func (n *SMTPNotifier) startTLS() error {
+	// Skips STARTTLS if is disabled in configuration.
+	if n.config.DisableStartTLS {
+		n.log.Warn("STARTTLS configuration is set to disable STARTTLS requirement (only useful for unauthenticated emails over plain text)")
```

Comment:
```suggestion
		n.log.Warn("Notifier SMTP connection has opportunistic STARTTLS explicitly disabled which means all emails will be sent insecurely over plain text and this setting is only necessary for non-compliant SMTP servers which advertise they support STARTTLS when they do not")
```

		n.log.Debugf("Notifier SMTP connection is already encrypted, skipping STARTTLS")

Reply:
Also maybe we should be logging this in the config validation or in the SMTP newup?